### PR TITLE
Eslintrc fix

### PR DESCRIPTION
**Pull request by Meamzy**
Installed prettier-eslint-config

In order to apply fix, you'll need to use 'npm install' on your local environment.

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by YuriiDubnytskyi — APPROVED** [review]

File: .prettierrc [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "useTabs": false,
-  "tabWidth": 2,
```
Comment (YuriiDubnytskyi) [inline .prettierrc:3]: Why did you delete this? Is it will work?

**Review by YuriiDubnytskyi — CHANGES_REQUESTED** [review]

**Review by sergklukvin — CHANGES_REQUESTED** [review]
Без імені

Why was the lock.json package changed?

**Review by sergklukvin — CHANGES_REQUESTED** [review]
Без імені1

Prettier isn't correct

**Comment by Meamzy** [comment]
Going to do some checks, thx for reviews.

**Pull request closed without merging**